api/logging_config.py

```diff
@@ -0,0 +1,40 @@
+"""
+Centralized logging configuration for the MOPED API
+"""
+
+import logging
+import os
+from typing import Optional
+
+
+def setup_logging(level: Optional[str] = None) -> logging.Logger:
```

Comment:
Super minor, but you can replace `None` with `"INFO"` here and remove the test down below